### PR TITLE
improve migration job throughput

### PR DESCRIPTION
**Pull request by rccrdpccl**
The aim of this PR is to improve migration job's throughput, thus decrease running time

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -184,9 +184,11 @@ objects:
           - "-c"
           - "ELASTIC_AUTH_HOST=https://${ELASTIC_USER}:${ELASTIC_PASSWORD}@${ELASTIC_HOST:8};\
    for index in ${MIGRATION_INDICES//,/ }; do \
+     aws s3 cp s3://${AWS_S3_BUCKET}/migration-v2_${index}.json .
```
Comment (rccrdpccl) [inline openshift/template.yaml:187]: We have many problems related to streaming this file with aws js client. Let's try to copy to filesystem first, we should solve connectivity issues, although the files are pretty big and we might incur in other problems

**Review by gamli75 — APPROVED** [review]

**Pull request merged**